applications/ParticleMechanicsApplication/python_scripts/deprecated_python_scripts/deprecated_particle_mpm_solver.py

```diff
@@ -0,0 +1,370 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications and dependencies
+import KratosMultiphysics.SolidMechanicsApplication as KratosSolid
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
+
+# Check that KratosMultiphysics was imported in the main script
+KratosMultiphysics.CheckForPreviousImport()
+
+def CreateSolver(grid_model_part, initial_material_model_part, material_model_part, custom_settings):
+    return ParticleMPMSolver(grid_model_part, initial_material_model_part, material_model_part, custom_settings)
+
+class ParticleMPMSolver(object):
+
+    ### Solver constructor
+    def __init__(self, grid_model_part, initial_material_model_part, material_model_part, custom_settings):
+        # Default settings
+        self.model_part1 = grid_model_part              #grid_model_part
+        self.model_part2 = initial_material_model_part  #initial_model_part
```

Comment:
Additionally the model part names are not very descriptive once you convert into member variables

Reply:
this is in the deprecated solver, @bodhinandach wont touch this